Makefile

```diff
@@ -69,7 +69,7 @@ FC = mpifort
 
 # Include a list of all the source files
 include $(CCX)/Makefile.inc
-SCCXMAIN = ccx_2.16.c
+SCCXMAIN = ccx_2.19.c
```

Comment:
There are a few more `2.16` further down (which actually don't matter, the number is not connected to the CalculiX version). Maybe it's a good chance to make this a variable?

Reply:
Probably good time for a variable yes.
Should we investigate the tutorial in the near future? (post release)

Reply:
I am investigating the tutorial at the moment, but this is independent of this PR.

Reply:
Related issue: https://github.com/precice/tutorials/issues/260